evaluation/dft/Makefile

```diff
@@ -0,0 +1,32 @@
+SRC := *.c
```

Comment:
Perhaps we can create a shared Makefile where all you have to change is the parameters for the specific benchmark. Don't have to deal with this just yet.

Reply:
https://github.com/cucapra/diospyros/issues/34